cmd/mothership/mothership/start/http.go

```diff
@@ -174,6 +184,46 @@ func callHandler(o *Options, handler func(o *Options, w http.ResponseWriter, r *
 	}
 }
 
+func deleteReconcilationsByCluster(o *Options, w http.ResponseWriter, r *http.Request) {
+	params := server.NewParams(r)
+	_, err := params.Int64(paramContractVersion)
```

Comment:
optional: you could use inline 'if' conditions if returned values scope is not required outside.
e.g. if _, err := params.Int64(paramContractVersion); err != nil {...}